sunpy/instr/goes.py

```diff
@@ -147,13 +160,13 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
     2014-01-01 00:00:06  7e-07  7e-06  11.28295376  4.78577516e+48
 
     """
-
     # Check that input argument is of correct type
     if not isinstance(goeslc, lightcurve.GOESLightCurve):
         raise TypeError("goeslc must be a GOESLightCurve object.")
```

Comment:
Good point.  Although if you just say input, might the user be confused as to which input?  At least this way when the user looks at the source code it is clear.  But it's not ideal.  The ideal would be that you can somehow store the name of variable input by the user and use that in the error message.  But I don't even know if that's possible.